tests/tests/eth_provider.rs

```diff
@@ -1420,3 +1420,13 @@ async fn test_transaction_by_hash(#[future] katana_empty: Katana, _setup: ()) {
         transaction.tx
     );
 }
+
+#[rstest]
+#[awt]
+#[tokio::test(flavor = "multi_thread")]
+async fn test_with_other_fields(#[future] katana: Katana, _setup: ()) {
+    let run_out_of_resources_receipt = katana.most_recent_run_out_of_resources_receipt().unwrap();
+
+    // Verify the receipt
+    assert_eq!(run_out_of_resources_receipt.other.get("isRunOutOfRessources"), Some(&serde_json::Value::Bool(true)));
+}
```

Comment:
Here this is valid but you never test the provider function to ensure that it returns the same result. You should also call this method from the provider 

https://github.com/kkrt-labs/kakarot-rpc/blob/a8117ee97586942f54ad2dd3cb6d2a3776ffcb71/src/eth_rpc/api/eth_api.rs#L106

to ensure that the result you obtain is the same

Reply:
fixed, thanks